sky/clouds/gcp.py

```diff
@@ -1034,3 +1109,9 @@ def delete_image(cls, image_id: str, region: Optional[str]) -> None:
             error_msg=f'Failed to delete image {image_name!r}',
             stderr=stderr,
             stream_logs=True)
+
+
+def reservation_self_link_to_name(self_link: str) -> str:
+    """Converts a reservation self link to a reservation name."""
+    parts = self_link.split('/')
+    return '/'.join(parts[-6:-4] + parts[-2:])
```

Comment:
Question: how does a user get the name for their reservation from the console or gcloud CLI? Can we just use the ending of the `self_link` as the name instead?
```
projects/skypilot-xxx/zones/us-central1-c/reservations/specific-reservation-test
```

Reply:
And no it is not possible to use the self_link :(